src/packages/documents/documents/repository/detail/document-detail.server.data-source.ts

```diff
@@ -26,6 +28,7 @@ export class UmbDocumentServerDataSource implements UmbDetailDataSource<UmbDocum
 	 */
 	constructor(host: UmbControllerHost) {
 		this.#host = host;
+		this.#languageRepository = new UmbLanguageCollectionRepository(host);
```

Comment:
Also, i'm not sure if we do this in other places, but it feels wrong that a Data-Source initialises another Repository. But this is really just a stomach feeling i got when reading this. :-)